terraform/restore-database/README.md

```diff
@@ -0,0 +1,41 @@
+# Restore a database
+
+This script creates a replica of an existing rds database from a snapshot. It also updates the database url parameter in the parameter store to point to the new   
+
+## Create the replica
+`export TF_VAR_database=consignment-api` or `export TF_VAR_database=keycloak`
+
+
+```
+export TF_VAR_tdr_account_number=xxxxxxxxxxx
+export DB_NO_DASH=$(echo $TF_VAR_database | sed  's/-//g')
+export TF_VAR_cluster_identifier=$(aws rds describe-db-clusters --query="DBClusters[?DatabaseName=='$DB_NO_DASH'].DBClusterIdentifier" --output text)
+export PROFILE=intg # replace with staging or prod if using those environments
+
+terraform init
+terraform apply
+```
+
+The restore may take 10-15 minutes depending on the size of the database when you run this.
```

Comment:
How do you know when it is finished? Does the Terraform script not complete until the restore has finished, or do you have to check on the console?

Reply:
Terraform will wait until the cluster is up. I've added in a line to say that.